src/helper.ts

```diff
@@ -0,0 +1,232 @@
+/*
```

Comment:
This comment style is inconsistent with the rest of the files.

Reply:
got it!